src/time/span.cr

```diff
@@ -84,70 +87,14 @@ struct Time::Span
     @nanoseconds = nanoseconds.to_i32
   end
 
-  def self.new(*, nanoseconds : Int)
```

Comment:
This overload must not be removed. There should be a constructor that allows passing nanoseconds with overflow.